src/app/SecurityPanel/Credentials/CreateJmxCredentialModal.tsx

```diff
@@ -74,23 +70,35 @@ export const CreateJmxCredentialModal: React.FunctionComponent<CreateJmxCredenti
       showClose={true}
       onClose={props.onClose}
       title="Store JMX Credentials"
-      description="Creates stored credentials for a given target. 
-        If a Target JVM requires JMX authentication, Cryostat will use stored credentials 
+      description="Creates stored credentials for a given target.
+        If a Target JVM requires JMX authentication, Cryostat will use stored credentials
         when attempting to open JMX connections to the target."
     >
-      <Form>
+      <JmxAuthForm onSave={onSave} onDismiss={props.onClose}>
         <FormGroup
-          label="Target"
+          label="Match Expression"
           isRequired
-          fieldId="target-select"
-          helperTextInvalid="Select a target"
-          validated={validTarget}
+          fieldId="match-expression"
+          helperText={
```

Comment:
Should the helper text be right below the text input for match expression tho? Here it shows the helper text at the end of the form?

![Screenshot from 2022-07-12 16-50-23](https://user-images.githubusercontent.com/68053619/178592808-0c62f8e6-e0a8-415a-9f77-58e6da380d5d.png)

Reply:
![image](https://user-images.githubusercontent.com/3787464/178740710-f68ea9ce-a206-4cca-843d-073cf6ab1f3f.png)

Reply:
Looks great to me :D